sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -155,15 +156,28 @@ export class ApplicationApi extends HttpBaseClient {
   }
 
   /**
-   * Get status of all enrollments of a student application.
-   * @param applicationId Student application.
-   * @returns application progress details.
+   * Get details for the application enrolment status of a student application.
+   * @param applicationId student application id.
+   * @returns details for the application enrolment status.
    */
-  async getApplicationEnrolmentDetails(
+  async getEnrolmentApplicationDetails(
     applicationId: number,
-  ): Promise<ApplicationCOEDetailsAPIOutDTO> {
-    return this.getCall<ApplicationCOEDetailsAPIOutDTO>(
+  ): Promise<EnrolmentApplicationDetailsAPIOutDTO> {
+    return this.getCall<EnrolmentApplicationDetailsAPIOutDTO>(
       this.addClientRoot(`application/${applicationId}/enrolment-details`),
     );
   }
+
+  /**
+   * Get details for an application on at completed status.
+   * @param applicationId application id.
+   * @returns details for an application on at completed status.
+   */
+  async getCompletedApplicationDetails(
+    applicationId: number,
+  ): Promise<CompletedApplicationDetailsAPIOutDTO> {
+    return this.getCall<CompletedApplicationDetailsAPIOutDTO>(
+      this.addClientRoot(`application/${applicationId}/completed-details`),
```

Comment:
do we need "details" in the api?

Reply:
Maybe we do maybe we do not.
Right now we have the API for `In progress` status, `Enrolment` status, `Completed` status, and "progress bar details", they are:
- Get(":applicationId/progress-details")
- Get(":applicationId/in-progress")
- Get(":applicationId/enrolment-details")
- Get(":applicationId/completed-details")
@dheepak-aot @andrepestana-aot @sh16011993, thoughts?

Reply:
@ann-aot are you referring to the name finishing with "details" in the URLs?

Reply:
I am fine either way. If the details make the URI look meaningful (e.g ":applicationId/completed-details" vs ":applicationId/completed") it can used IMO.

Reply:
yes @andrepestana-aot

Reply:
Changed as.
- Get(":applicationId/in-progress")
- Get(":applicationId/enrolment")
- Get(":applicationId/completed")